frontend/fingertips-frontend/components/viewPlots/OneIndicatorTwoOrMoreAreasViewPlots/OneIndicatorTwoOrMoreAreasViewPlots.test.tsx

```diff
@@ -192,28 +196,33 @@ describe('OneIndicatorTwoOrMoreAreasViewPlots', () => {
     });
   });
 
-  it('should render the ThematicMap when all areas in a group are selected', async () => {
-    const searchState: SearchStateParams = {
-      [SearchParams.GroupAreaSelected]: 'ALL',
-      [SearchParams.AreaTypeSelected]: 'regions',
-      //  DHSCFT-483 to remove this when using GAS
-      [SearchParams.GroupSelected]: 'a group',
-    };
+  describe('ThematicMap', () => {
+    it('should render the ThematicMap when all areas in a group are selected', async () => {
+      const searchState: SearchStateParams = {
+        [SearchParams.GroupAreaSelected]: 'ALL',
+        [SearchParams.AreaTypeSelected]: 'regions',
+        //  DHSCFT-483 to remove this when using GAS
+        [SearchParams.GroupSelected]: 'a group',
+      };
 
-    render(
-      <OneIndicatorTwoOrMoreAreasViewPlots
-        healthIndicatorData={[
-          mockHealthData[108][1],
-          mockHealthData[108][2],
-          mockHealthData[108][3],
-        ]}
-        searchState={searchState}
-        mapData={mockMapData}
-        areaCodes={mockAreas}
-      />
-    );
-    expect(
-      await screen.findByTestId('thematicMap-component')
-    ).toBeInTheDocument();
+      render(
+        <OneIndicatorTwoOrMoreAreasViewPlots
+          healthIndicatorData={[
+            mockHealthData[108][1],
+            mockHealthData[108][2],
+            mockHealthData[108][3],
+          ]}
+          searchState={searchState}
+          mapData={mockMapData}
+          areaCodes={mockAreas}
+        />
+      );
+      expect(
+        await screen.findByTestId('thematicMap-component')
+      ).toBeInTheDocument();
+    });
+
+    // DHSCFT-483 to add test
+    // it('should render the ThematicMap when all areas in a group are selected'...
```

Comment:
This might be a good place to use `it.todo` ([docs](https://jestjs.io/docs/api#testtodoname)). It'll highlight the test as needing implementing, so it's harder for us to miss in future. I don't think it'll cause the overall test run to fail though (although I may be misremembering).

Also, I think this is intended to be a `should not render the ThematicMap when not all areas in a group are selected` test, isn't it?
```suggestion
    // it('should not render the ThematicMap when not all areas in a group are selected'...
```